composer.json

```diff
@@ -3,14 +3,16 @@
     "description": "A multi-language port of Browserscope's user agent parser.",
     "require": {
         "php": "^7.2",
-        "composer/ca-bundle": "^1.1"
+        "composer/ca-bundle": "^1.1",
+        "phpstan/phpstan": "^0.12.33"
```

Comment:
Is there a reason to add this not as Dev-Dependency?

Reply:
Yes! The reason is I made a mistake. Thanks for spotting 👍